lib/ssl/src/tls_handshake.erl

```diff
@@ -476,9 +476,9 @@ decode_handshake(Version, Tag, Msg) ->
     ssl_handshake:decode_handshake(Version, Tag, Msg).
 
 
-ocsp_expect(true) ->
+default_stapling_status(true) ->
     staple;
```

Comment:
`staple` meaning is not obvious and can be easily confused with `stapled`.
consider renaming atom to something better.

Reply:
changed to `received_peer` which is supposed to mean, _staple(OCSP response) received for peer certificate_
9e5f3ce3ae